devops/containers/ubuntu2204_preinstalled.Dockerfile

```diff
@@ -5,8 +5,7 @@ FROM $base_image:$base_tag
 
 COPY scripts/drivers_entrypoint.sh /drivers_entrypoint.sh
 RUN mkdir -p /opt/sycl
-ADD llvm_sycl.tar.zst /tmp
-RUN tar -I 'zstd -d' -xf /tmp/llvm_sycl.tar.zst -C /opt/sycl && rm /tmp/llvm_sycl.tar.zst
+ADD sycl_linux.tar.gz /tmp
 
```

Comment:
Why did you remove installation to /opt/sycl?

Reply:
See latest commit's message: 

> [Remove unpack - "ADD *.tar.gz /tmp" does it automatically](https://github.com/intel/llvm/pull/10560/commits/2ab670bca7c32bb92fbcd4cf9cadc7b73e71b2af)

It also restores it to what we had before `zstd` compression was introduced in https://github.com/intel/llvm/pull/9675/files#diff-e50cbabbef9e5364e1fbfe55bd3c6385f327ccceeab0e45674913f08afbfb05dR9

Reply:
Are you saying that "ADD *.tar.gz /tmp" does "tar -I 'gunzip' -xf /tmp/sycl_linux.tar.gz -C /opt/sycl && rm /tmp/sycl_linux.tar.gz"? I don't think so.

Reply:
Almost: https://github.com/intel/llvm/actions/runs/5684595337/job/15407779458

I have to change `/tmp` to `/opt/sycl` but the `tar xf` and `rm` part is automatic. Done in the last commit.